### PR TITLE
Add AppVeyor config

### PR DESCRIPTION
**Pull request by DanRStevens**

### CODE REVIEW

**Comment by DanRStevens** [comment]
Looks like the x64 configurations does not have the necessary:
```
<AdditionalIncludeDirectories>OP2Utility\include;FreeImage\Dist\x32;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

**Comment by DanRStevens** [comment]
@Brett208: Might be good if you checked the project setting updates on this one.

File: OP2MapImager.vcxproj [hunk 1]
```diff
@@ -203,9 +211,13 @@ if $(ConfigurationName) == Release (
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <PreprocessorDefinitions>NDEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>OP2Utility\include;FreeImage\Dist\x64;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <SDLCheck>false</SDLCheck>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
```
Comment (Brett208) [inline OP2MapImager.vcxproj]: This property is also added in open branch #20. Maybe we should merge that branch. I was going to wait until we had finalized the change list, but maybe better to merge now.

File: OP2MapImager.vcxproj [hunk 2]
```diff
@@ -137,8 +139,12 @@ if $(ConfigurationName) == Release (
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>OP2Utility\include;FreeImage\Dist\x64;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```
Comment (Brett208) [inline OP2MapImager.vcxproj]: This property is also added in open branch #20. Maybe we should merge that branch. I was going to wait until we had finalized the change list, but maybe better to merge now.

**Comment by DanRStevens** [comment]
I noticed the 32-bit FreeImage.lib file was added to the project. I believe that's why it didn't need to be added as an additional linker dependency in the project settings.

**Pull request merged**